share/spice/leak_db/leak_db.js

```diff
@@ -12,7 +12,8 @@
                         query : api_result.type + ' hash';
 
         api_result.hashes = api_result.hashes[0];
-        
+        delete api_result.hashes.plaintext;
```

Comment:
I realize now that plaintext is used for showing what the plaintext version of a password is, when the user enters a hash as the lookup.

E.g. 
- "leakdb MySuperNicePassword" => MD5: 8ea6794b65f326b437bd2d562ea10700
- "leakdb 8ea6794b65f326b437bd2d562ea10700" => Plaintext: MySuperNicePassword